packages/react/src/components/Button/Button.js

```diff
@@ -42,6 +45,13 @@ const Button = React.forwardRef(function Button(
     [`${prefix}--btn--disabled`]: disabled,
   });
 
+  if (__DEV__ && small) {
```

Comment:
Would this be:

```js
  if (__DEV__) {
    warning(
     // First argument, if true, will log out the warning message
     small !== undefined,
      // Slight tweak to message to match other deprecation warnings
      `The Button prop \`small\` has been deprecated in favor of \`size\`. ` + 
      `Please use the \`size\` prop instead and set it to \`small\`.`
    );
  }
```
?

Reply:
@joshblack earlier i wasn't getting expected results from the test with exact match operator, had to loosen it up for sniff for truthy values. will test this reorg tho and see if it helps!

Reply:
updating to:

Reply:
updating to:

```
if (__DEV__) {
    warning(
      !small,
```

(fishing for truthy values on purpose here, but lmk if that's out of line!)